src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/utils/ResponseEntityBuilder.java

```diff
@@ -3,13 +3,12 @@
 import org.slf4j.Logger;
 import org.springframework.http.CacheControl;
 import org.springframework.http.ResponseEntity;
-import uk.gov.hmcts.reform.wataskmanagementapi.controllers.TaskActionsController;
 
 import static org.slf4j.LoggerFactory.getLogger;
 
 public final class ResponseEntityBuilder {
 
-    private static final Logger LOG = getLogger(TaskActionsController.class);
+    private static final Logger LOG = getLogger(ResponseEntityBuilder.class);
```

Comment:
Maybe it could be a little cleaner of we used @Slf4j lombok annotation instead?  
https://github.com/hmcts/wa-task-management-api/blob/master/src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/TaskSearchQueryBuilder.java#L31